typescript/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "aiconfig",
-  "version": "1.0.5",
+  "version": "1.0.7",
```

Comment:
Why not 1.0.6?

Reply:
1.0.6 is the current public release version.  It wasn't updated in the repo.

Updated readme for clarity